georocket-cli/src/main/java/io/georocket/GeoRocketCli.java

```diff
@@ -159,6 +147,25 @@ public static void main(String[] args) throws IOException {
     }
   }
 
+  /**
+   * Setup GeoRocket Cli.
+   */
+  protected static void setup() {
```

Comment:
I do not overwrite this method, I just call it. Nevertheless, I think your way is better coding. Thanks.